docs/plugins.rst

```diff
@@ -24,8 +24,7 @@ Airflow has a simple plugin manager built-in that can integrate external
 features to its core by simply dropping files in your
 ``$AIRFLOW_HOME/plugins`` folder.
 
-The python modules in the ``plugins`` folder get imported,
```

Comment:
Could you add a [warning](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-warning) here that operators and Hooks should to be packaged + link to docs about module loading? I know this causes a lot of problems for users, and they can easily overlook this behavioral change.

Reply:
They don't need to be packaged at all -- plain importable python modules in plugins/ folder or as https://github.com/apache/airflow/blob/master/docs/howto/custom-operator.rst shows continue to work just fine.

Reply:
I have added a note to updating.md though, I didn't have one for Hooks.

Reply:
That's right, but most users use `airflow.operators. *` or` airflow.hooks. *`, because that's how it was described on this page before and we didn't have a better documentation that covered loading a module in detail. 
A small note, since Airflow 2.0, loading modules via `airflow. {operators,, hooks, sensors}. *` Is not supported. Many people can help if they have read archival materials and guides on the Internet before. Additionally, it is worth giving a link to the docs about [module loading](https://airflow.readthedocs.io/en/latest/modules_management.html) one and an example.

In line. with the annual survey, over 60% of people misused plugins. See: https://airflow.apache.org/blog/airflow-survey/

Reply:
> .. versionchanged:: 2.0
> 
>    Importing operators, sensors, hooks added in plugins via `airflow.{operators,sensors,hooks}.<plugin_name>` is no longer supported, and these extensions should just be imported as regular python modules.
> 
>    ```
>    from airflow.hooks.my_plugin import MyHook
>    ```
> 
>    You should instead import it as:
> 
>    ```
>    from my_plugin import MyHook
>    ```
> 
>    It is still possible (but not required) to "register" hooks in plugins. This is to allow future support    for dynamically populating the Connections form in the UI.
> 
>    For more information, see: :doc:`/modules_management.html` and :doc:`/howto/custom-operator.html`

Reply:
Nested code/blocks didn't render right in `.. versionchanged::` so I've shortened it. (sphinx seems to expect `versionchanged` to be "short")

This added to just at the end of this section.

![image](https://user-images.githubusercontent.com/34150/98364351-8bbafc00-2028-11eb-875b-ca0132a74c70.png)